core/src/deno/script.rs

```diff
@@ -0,0 +1,147 @@
+use anyhow::Result;
+use deno_core::{op, Extension, JsRuntime, OpState, ZeroCopyBuf};
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+use std::borrow::Cow;
+use std::rc::Rc;
+use std::{thread, time::Duration};
+
+pub struct Script {
+    runtime: JsRuntime,
+    last_rid: u32,
+    timeout: Option<Duration>,
+}
+
+impl Script {
```

Comment:
This is mostly adapted from `js_sandbox` directly + fixes to bump deno_core to 0.181